plugins/defaultplugins/common/model/srv6/srv6.proto

```diff
@@ -0,0 +1,95 @@
+syntax = "proto3";
+
+package srv6;
+
+message LocalSID {          /* SID(IPv6 Address) is part of key and NOT part of value(message) in ETCD key-value store */
```

Comment:
To be consistent with rest of the models, all the data used for southbound (VPP) should be stored inside the model, in this case SID.

Reply:
fixed in ea181a4262e68973038f769dcf05f0c9cc682053